tests/utils.py

```diff
@@ -831,7 +833,7 @@ def get_classifier_bb(defences=None):
     # define blackbox classifier
     def predict(x):
         with open(
-            os.path.join(os.path.dirname(os.path.dirname(__file__)), "data/mnist", "api_output.txt")
+                os.path.join(os.path.dirname(os.path.dirname(__file__)), "data/mnist", "api_output.txt")
```

Comment:
Please don't add indent, this is following Black style.
```suggestion
            os.path.join(os.path.dirname(os.path.dirname(__file__)), "data/mnist", "api_output.txt")
```